controllers/backplaneconfig_controller.go

```diff
@@ -696,12 +709,15 @@ func (r *MultiClusterEngineReconciler) ensureCustomResources(ctx context.Context
 	return ctrl.Result{}, nil
 }
 
-func (r *MultiClusterEngineReconciler) finalizeBackplaneConfig(ctx context.Context, backplaneConfig *backplanev1.MultiClusterEngine) error {
+func (r *MultiClusterEngineReconciler) finalizeBackplaneConfig(ctx context.Context, backplaneConfig *backplanev1.MultiClusterEngine, ocpConsole bool) error {
 	log := log.FromContext(ctx)
-	_, err := r.removePluginFromConsoleResource(ctx, backplaneConfig)
-	if err != nil {
-		log.Info("Error ensuring plugin is removed from console resource")
-		return err
+	var err error
+	if ocpConsole {
```

Comment:
There isn't a need to add an if statement here. The plugin should be removed if present no matter what.